funcs_utils.php

```diff
@@ -46,85 +48,6 @@ function write_file($path, $contents)
     info("Wrote to $path");
 }
 
-/**
- * Returns all the supported modules for a particular cms major version
- * Will download the list if it doesn't exist
- */
-function supported_modules($cmsMajor)
-{
-    $filename = "_data/modules-cms$cmsMajor.json";
-    if (!file_exists($filename)) {
-        $url = "https://raw.githubusercontent.com/silverstripe/supported-modules/$cmsMajor/modules.json";
-        info("Downloading $url to $filename");
-        $contents = file_get_contents($url);
-        file_put_contents($filename, $contents);
-    }
-    $json = json_decode(file_get_contents($filename), true);
-    if (is_null($json)) {
-        $lastError = json_last_error();
-        error("Could not parse from $filename - last error was $lastError");
-    }
-    $modules = [];
-    foreach ($json as $module) {
-        $ghrepo = $module['github'];
-        $modules[] = [
-            'ghrepo' => $ghrepo,
-            'account' => explode('/', $ghrepo)[0],
-            'repo' => explode('/', $ghrepo)[1],
-            'cloneUrl' => "git@github.com:$ghrepo.git",
-        ];
-    }
-    return $modules;
-}
-
-/**
- * Hardcoded list of non-supported, additional repositories to standardise (e.g. silverstripe/gha-*)
- *
- * Repositories in this list should only have a single supported major version
- * This will only be included if the $cmsMajor is the CURRENT_CMS_MAJOR
- */
-function extra_repositories()
-{
-    $importantRepos = [
-        'silverstripe/markdown-php-codesniffer',
-        'silverstripe/silverstripe-standards',
-        'silverstripe/documentation-lint',
-        'silverstripe/.github',
-    ];
```

Comment:
The set of "extra" repos we're gonna end up with after this change is a lot bigger than we were getting before. I've added a note to https://github.com/silverstripe/.github/issues/242 that when this gets run next we should take care to check if there's anything we want to skip and add an exclude list if there is.